### PR TITLE
Hinxtonconference Blog Post

### PR DESCRIPTION
**Pull request by EngyNasr**
Hinxton ABPHM conference Blog Post

### CODE REVIEW

File: content/news/2023-05-06-ABPHM/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: Hinxton, UK
+tags: [conference, talk, bioinformatics showcase, pathogen detection]
+authors: engynasr
+supporters:
+- microbial genomics
+components: true
+subsites: [all-eu]
+---
+
+From the 3rd to the 5th of May, Engy Nasr from the Freiburg Galaxy team travelled to Hinxton, England to join the [**Applied Bioinformatics & Public Health Microbiology 2023**](https://coursesandconferences.wellcomeconnectingscience.org/event/applied-bioinformatics-public-health-microbiology-20230503/).
```
Comment (bgruening) [inline content/news/2023-05-06-ABPHM/index.md:15]: From ... Engy presented her work about ... at the ...

File: content/news/2023-05-06-ABPHM/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<twitter tweet="1653738329647595521"></twitter>
+<twitter tweet="1654050208056049665"></twitter>
+
+The 9th Applied Bioinformatics and Public Health Microbiology conference, which took place in the wellcome connecting campus, provided a multidisciplinary forum to demonstrate how advances in microbial and viral genomics, bioinformatics, data science, and sequencing technology are being used to meet the needs of public health.
```
Comment (bgruening) [inline content/news/2023-05-06-ABPHM/index.md:20]: Isn't it the "Wellcome Genome Campus" ?

File: content/news/2023-05-06-ABPHM/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+On the first day; there were two talks sessions followed by a poster session. 111 posters were presented on the first and the second day. The first talks session was about new approches in bioinformatics and the second was about genomic epidemiology in the field.
+
+On the second day; we had three talks sessions; two of them were about the public health and the third was about metagenomics for survellance. In between we had a bioinformatics showcase where I presented the Pathogen Detection workflow we created in Galaxy along with its [**training material**](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html). There were 16 abstracts presented in the bioinformatics showcase, and we won the award as the best abstract presented.
```
Comment (bgruening) [inline content/news/2023-05-06-ABPHM/index.md:30]: Where Engy presented?

File: content/news/2023-05-06-ABPHM/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+<twitter tweet="1654400374583746561"></twitter>
+
+On the third and last day; we had a panel discussion regarding preparing for the next pandemic then we had a closing session where Chikwe Ihekweazu from the [**WHO Hub for Pandemic and Epidemic Intelligence**](https://pandemichub.who.int/) gave the final talk annoncing the launching of the [**International Paqthogen Surveillance Network (IPSN)**](file:///home/nasr/Downloads/9789240046979-eng-1.pdf) on the 20th of May 2023
+
+<twitter tweet="1654485708378128388"></twitter>
+
+Overall, I have really enjoyed attending the conference. I met a lot of scientist working on similar projects where we had a chance to exchange our knowledge. I would like to thank the Microbial Genomics for choosing my abstract to win the award and the Wellcome genome campus of having me attenging the conference. Wish to see all the people I met in 2 years in the 10th Applied Bioinformatics and Public Health Microbiology conference.
```
Comment (bgruening) [inline content/news/2023-05-06-ABPHM/index.md:38]: So either "I" or in third person, but I guess consistency is good.

**Review by bgruening — COMMENTED** [review]
Why not write about your talk? Can you link your slides?
Any interesting outcome you would like to inform the Galaxy community?

File: content/news/2023-05-06-ABPHM/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Galaxy taking part @ Applied Bioinformatics & Public Health Microbiology 2023
+date: '2023-05-06'
+tease: "From the 3rd to the 5th of May, Engy Nasr from the Freiburg Galaxy team travelled to Hinxton, England to join and present her work at the Applied Bioinformatics & Public Health Microbiology 2023 conference"
+location:
+  name: Hinxton, UK
+tags: [conference, talk, bioinformatics showcase, pathogen detection]
+authors: engynasr
+supporters:
+- microbial genomics
```
Comment (bgruening) [inline content/news/2023-05-06-ABPHM/index.md:10]: @bebatut what supportes should we add here? Any idea?

Comment (bebatut) [reply content/news/2023-05-06-ABPHM/index.md:10]: EOSC-Life for sure. Maybe ELIXIR too

**Pull request merged**